api/kas-api.yaml

```diff
@@ -170,11 +170,11 @@ components:
             oneOf:
               - type: object
                 required:
-                  - clientPayloadSignature
+                  - signedRequestToken
```

Comment:
To be a PATCH this should deprecate the clientPayloadSignature, not replace it, otherwise this is arguably a major revision (clients with the same major revision should be backwards compatible)

You can say Deprecated, servers MAY allow this request when used, subject to configuration, instead of MUST

Reply:
@dmihalcik-virtru The OpenAPI spec is an example - not part of the spec. I should move it to `examples`.

This is not intended to say your services MUST have a specific `clientPayloadSignature` or `signedRequestToken` key in their requests, merely that something _like_ that must exist - how and where is up to the implementer.

Current openTDF implementations all use `signedRequestToken` already, so that's why I didn't make it a patch.

The need for _a request signature_ was previously introduced as part of the V4 major spec bump.